Mocha/palette

```diff
@@ -0,0 +1 @@
+['#45475a', '#f38ba8', '#a6e3a1', '#f9e2af', '#89b4fa', '#cba6f7', '#f5c2e7', '#cdd6f4', '#6c7086', '#f38ba8', '#a6e3a1', '#f9e2af', '#89b4fa', '#cba6f7', '#f5c2e7', '#cdd6f4']
```

Comment:
according to https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md#regular-colours, shouldn't the color scheme be this:
['#45475A', '#E0819D', '#8CBD8A', '#E5D1A3', '#7498D2', '#CDA6C5', '#88D0C5', '#BCC4E0', '#525568', '#CB7992', '#A7E6A2', '#E5D1A3', '#7498D2', '#FCC7EE', '#92E0D3', '#A7AFCA']